src/next/azext_next/utils.py

```diff
@@ -177,15 +240,16 @@ def get_last_exception(cmd, latest_command):
     return ''
 
 
-def get_title_case(str):
-    if not str:
-        return str
-    str = str.strip()
-    return str[0].upper() + str[1:]
+def get_title_case(s):
+    if not s:
+        return s
+    s = s.strip()
+    return s[0].upper() + s[1:]
```

Comment:
Generally, please do not use a single character as the variable name, which will make the code difficult to understand